code-challenges.test.js

```diff
@@ -43,26 +33,36 @@
 
 // b) Create the function that makes the test pass.
 // Psuedo Code:
-// For this I need to create a function that takes in the variable "fibonacciSequence"
-// I'm going to utilize the for loop method to add up the numbers
-// I'll create two extra variables of 0 and 1 
-
+// Create a function called fibonacciSequence with a num param
+// Create a variable with an array of 1,1 to hold the first two values of the Fibonacci Sequence
+// Create a for loop so the function will increase with each iteration
+// Use .push to push the index of the array variable and + fSArray[i -1] 
+// return fSArray
```

Comment:
Excellent work!  Logic looks great and pseudocode is a great representation of how to get there!